downstairs/src/lib.rs

```diff
@@ -2539,7 +2552,6 @@ impl Downstairs {
                 // working on outstanding jobs, or a way to merge. But for now,
                 // we just throw out what we have and let the upstairs resend
                 // anything to us that it did not get an ACK for.
-                work.clear();
```

Comment:
We no longer need the clear here?  Or, did we never really need it to begin with?

Reply:
The `clear` isn't (and was never) doing anything.

At the top of this conditional, we removed (not borrowed!) an `ActiveUpstairs` from the map
```rust
            if let Some(active_upstairs) = self
                .active_upstairs
                .remove(&upstairs_connection.upstairs_id)
            {
```

This `active_upstairs` is dropped at the end of the logical scope (i.e. right here); clearing the work map doesn't accomplish anything, because it's about to be dropped, and no one else can have ownership of it.

Reply:
Yes, thanks.  Thats what I thought, but wanted to be sure.  Having two independent writers both sending
jobs to a region would be, as I'm sure you know, a total disaster.  Thanks for clarifying.